src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.CitizenDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.Citizen;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    BetDao betDao;
+    @Inject
+    CitizenDao citizenDao;
+
+    public void handleCitizen() {
+        System.out.println("Please, add name and age! ");
```

Comment:
I would add a little instruction about input data format for the user to avoid unnecessary mistakes :)